packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx

```diff
@@ -174,23 +176,25 @@ const TableCalculationModal: FC<Props> = ({
                             placeholder={SQL_PLACEHOLDER}
                         />
                     </TableCalculationSqlInputWrapper>
+                    <div className={Classes.DIALOG_FOOTER}>
```

Comment:
I don't think we need to change this to fix the issue. The css change should be enough.

Reply:
Yeah i know, i just thought it made sense, this was living outside the dialog, and i thought it should be part of it. but i can move it out again :)